internal/modifier/cdi.go

```diff
@@ -92,7 +92,7 @@ func getDevicesFromSpec(logger logger.Interface, ociSpec oci.Spec, cfg *config.C
 
 	var devices []string
 	seen := make(map[string]bool)
-	for _, name := range container.VisibleDevicesFromEnvVar() {
+	for _, name := range container.VisibleDevices() {
```

Comment:
Please note that we already handle `CDIDevicesFromMounts` above, meaning that we may be duplicating logic here. Let's rather not make this change as part of this changeset.

Reply:
That would mean not making VisibleDevicesFromEnvVar private for now

Reply:
Sure. That comment was more of a todo